### PR TITLE
ESC-234

### PR DESCRIPTION
**Pull request by sam-miele**

### CODE REVIEW

File: app/uk/gov/hmrc/eusubsidycompliance/connectors/EisConnector.scala [hunk 1]
```diff
@@ -122,4 +125,38 @@ class EisConnector @Inject()(
...
+                       ): Future[UndertakingSubsidies] = {
+    //import uk.gov.hmrc.eusubsidycompliance.models.UndertakingSubsidies.format
+    import uk.gov.hmrc.eusubsidycompliance.models.json.eis.eisRetrieveUndertakingSubsidiesResponse
+    import uk.gov.hmrc.eusubsidycompliance.models.json.eis.eisRetrieveUndertakingSubsidiesResponseWrite
+
+    desPost[SubsidyRetrieve, UndertakingSubsidies](
+      s"$eisURL/$retrieveSubsidyPath",
+      SubsidyRetrieve(
+        ref,
+        Some((LocalDate.of(2000, 1, 1), LocalDate.now()))
```
Comment (sam-miele) [inline app/uk/gov/hmrc/eusubsidycompliance/connectors/EisConnector.scala:157]: I need to test 'None' as the default with upcoming stub changes

**Review by priyankadhiman3006 — APPROVED** [review]
